docs/topics/request-response.rst

```diff
@@ -607,6 +606,12 @@ Response objects
         This represents the :class:`Request` that generated this response.
     :type request: :class:`Request` object
 
+    :param address: The IP address of the server from which the Response originated.
+    :type address: :class:`twisted.internet.address.IPv4Address` object
```

Comment:
Hey @elacuesta and @Gallaecio! Why do you prefer address to ip_address, or server_ip_address? :) I liked previous name more, because there can be different meaning of response.address: URL address, company address string extracted from the response content, some address in memory, etc.

Reply:
`ip_address` works for me.